modules/core/src/passes/pick-layers-pass.ts

```diff
@@ -140,7 +133,7 @@ export default class PickLayersPass extends LayersPass {
   protected getLayerParameters(layer: Layer, layerIndex: number, viewport: Viewport): any {
     const pickParameters = {...layer.props.parameters};
 
-    if (!this._colors) {
+    if (!this._colors || layer.props.operation.includes(OPERATION.TERRAIN)) {
```

Comment:
If we type the prop properly as `Operation` (see above) we can just do
```suggestion
    if (!this._colors || layer.props.operation.includes('terrain')) {
```